Assets/Code/Scripts/LootBag.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LootBag : MonoBehaviour
+{
+    
+    public GameObject droppedItemPrefab;
+    public List<Loot> lootList = new List<Loot>();
+
+
+    List<Loot> GetDroppedItems()
+    {
+        int randomNumber = Random.Range(1,101); // 1-100
```

Comment:
Jakbyś podmienił w Loot jednak na float to pamiętaj żeby to tu też dostosować

Reply:
Zmienione na float